src/services/geolocation.js

```diff
@@ -1,137 +0,0 @@
-var q = require('q');
-var $ = require('jquery');
-var callbacks = $.Callbacks('unique');
-var latestCoordinates;
-
-var DEFAULTS = {
-	enableHighAccuracy: true, 
-	maximumAge        : 30000, 
-	timeout           : 27000
-};
-
```

Comment:
Please point me towards other places in the code where this service is used. I am looking for the places where the geodata is consumed.